Release.toml

```
@@ -212,4 +212,6 @@ version = "1.15.0"
     "migrate_v1.14.0_public-control-container-v0-7-2.lz4",
 ]
 "(1.14.0, 1.14.1)" = []
-"(1.14.1, 1.15.0)" = []
+"(1.14.1, 1.15.0)" = [
+    "migrate_v1.15.0_oci-defaults-resource-setting.lz4",
+]
```

after rebase, this now needs to be:
```suggestion
"(1.14.2, 1.15.0)" = [
    "migrate_v1.15.0_oci-defaults-resource-setting.lz4",
]
```